state-chain/pallets/cf-trading-strategy/src/tests.rs

```diff
@@ -32,16 +32,12 @@ const QUOTE_ASSET: Asset = cf_primitives::STABLE_ASSET;
 const BASE_AMOUNT: AssetAmount = 100_000;
 const QUOTE_AMOUNT: AssetAmount = 50_000;
 
-const BUY_TICK: Tick = -1;
-const SELL_TICK: Tick = 1;
+const SPREAD: Tick = 1;
```

Comment:
```suggestion
const SPREAD_TICK: Tick = 1;
```

Otherwise it's easy to get confused between the ticks `(-1, 1)` and the spread (`2` ticks).